server/handlers/meshery_pattern_handler.go

```diff
@@ -278,14 +278,14 @@ func (h *Handler) GetMesheryPatternsHandler(
 		return
 	}
 
-	token, err := provider.GetProviderToken(r)
+	// token, err := provider.GetProviderToken(r)
 	if err != nil {
 		http.Error(rw, "failed to get user token", http.StatusInternalServerError)
 		return
 	}
-	mc := NewContentModifier(token, provider, prefObj, user.UserID)
-	//acts like a middleware, modifying the bytes lazily just before sending them back
-	err = mc.AddMetadataForPatterns(r.Context(), &resp)
+	// mc := NewContentModifier(token, provider, prefObj, user.UserID)
+	// //acts like a middleware, modifying the bytes lazily just before sending them back
+	// err = mc.AddMetadataForPatterns(r.Context(), &resp)
 	if err != nil {
 		fmt.Println("Could not add metadata about pattern's current support ", err.Error())
```

Comment:
Yikes `fmt.Println`

Reply:
@Revolyssup will you please add comments that describe the behavior that these lines have (i.e. dry run)?